src/features/CurrentSessionView/currentSessionSummary/Guest.js

```diff
@@ -119,6 +121,20 @@ const Guest = ({ guest }) => {
       })
       .catch((error) => {
         console.log('Error denying the guest: ', error);
+      })
```

Comment:
Would not adding the guest data to database until after they are accepted into session saved some of this. I think it's fine, just wondering 👍

Reply:
Yes I originally wanted to do that as well. But currently both 'deny' a guest and 'remove' a guest invoke this `handleDenyGuest` function, and 'remove' a guest can only happen after the guest is allowed first. So if db is only updated on the 'allow' action, then we need to add a condition here to check whether a transaction exist before the deletion in case the guest is denied.

but yes, a few lines in `GuestQR.js` can be saved.